### DISCUSSION TITLE
Missing translation strings

### DISCUSSION THREAD

**Discussion by marciozomb13** [QL-Win/QuickLook#1105] [Q&A] [answered]
Hi developer, i wonder if you are aware of some missing strings on app UI, could you please check if they are hardcoded?
What i found so far:

"Stay on Top"
"Prevent Closing"
"Open With"
"MIME Type"
"Image Height"
"Image Width"

And the entire plugin installation screen.

Thank you for your time.

**Comment by rabelux** [comment, marked as answer]
He's aware of it and just too lazy to do it. If you're willing to contribute it would be very welcomed. See #868 for reference.

**Reply by marciozomb13** [reply]
Unfortunately I'm just a translator, don't have programming skills, otherwise i would gladly fix it.
